SUPPORT.md

```diff
@@ -0,0 +1,22 @@
+# Getting help
+
+The SuperBOL Studio OSS extension for Visual Studio Code is brought to you by the SuperBOL team (https://superbol.eu/) at OCamlPro (https://ocamlpro.com/).
+The goal of SuperBOL, in general, is to provide a complete open-source environment for COBOL programs to migrate to, based on the GnuCOBOL free and open-source compiler.
+
+## SuperBOL Studio OSS
+
+If you encounter any issue with this extension, do not hesitate to:
+
+- post [an issue](https://github.com/OCamlPro/superbol-studio-oss/issues/);
+
+- ask questions in the #superbol stream at https://chat.ocamlpro.com/.
+
+## GnuCOBOL
+
+The people at https://superbol.eu/ also contribute to GnuCOBOL, which is the tool that is used by the extension to compile and execute your COBOL programs.
+
```

Comment:
I'd like to see a small sentence at the end of that paragraph about community based support for the compiler here, maybe linking to https://sourceforge.net/projects/gnucobol/support